src/components/biggive-main-menu/biggive-main-menu.tsx

```diff
@@ -153,21 +167,31 @@ export class BiggiveMainMenu {
       </Host>
     );
 
-    /**
-     * @todo take prop passed in for base domain or environment (e.g. dev|staging|regression|production ) so we can
-     *       output appropriate nav links to stay in same envirionment.
-     */
     const presetContentMenu = () => {
+      if (!this.blogUrlPrefix || !this.blogUrlPrefix.endsWith('/')) {
+        throw new Error('Blog URL prefix must be set and end with /');
+      }
+
+      if (!this.donateUrlPrefix || !this.donateUrlPrefix.endsWith('/')) {
+        throw new Error('Donate URL prefix must be set and end with /');
```

Comment:
@bdsl I'm not sure about this. I think that when we want to link to the base of an app like Donate it looks a bit messier, and it also diverges from how we set base URIs in almost all existing apps.

Reply:
@NoelLH OK - happy to go with what you think fits better. Would you just change lines like

```
                              <a href={this.blogUrlPrefix + 'green-match-fund'} class="icon-green-match">
```

to include the / as
```
                              <a href={this.blogUrlPrefix + '/green-match-fund'} class="icon-green-match">
```

Maybe even better if I just make a function to combine the prefix with the slash and the specific URL then it can't be forgotten anywhere.

Reply:
@NoelLH fixed in f4b44bd